wikimedia/wikipedia-ios
# Fetch data from new Talk Page API

**Phabricator:** https://phabricator.wikimedia.org/T311071

### Notes
* This PR adds a class to fetch data from the new mobile Talk Page API
* It also renames the existing TalkPage API-related files

### Test Steps
1. You can add a call to this class from the Explore feed to trigger this method
2. check with article talk pages and user talk pages; check if the response does not throw any errors and parses correctly
